code/__DEFINES/dcs/signals/signals_heretic.dm

```diff
@@ -0,0 +1,18 @@
+/// Heretic signals
+/// From /obj/item/clothing/mask/madness_mask/process : (amount)
+#define COMSIG_HERETIC_MASK_ACT "void_mask_act"
+
+/// From /obj/item/melee/touch_attack/mansus_fist/on_mob_hit : (mob/living/source, mob/living/target)
+#define COMSIG_HERETIC_MANSUS_GRASP_ATTACK "mansus_grasp_attack"
+/// Default behavior is to use the hand, so return this to blocks the mansus fist from being consumed after use.
+	#define COMPONENT_BLOCK_HAND_USE (1<<0)
+/// From /obj/item/melee/touch_attack/mansus_fist/afterattack_secondary : (mob/living/source, atom/target)
+#define COMSIG_HERETIC_MANSUS_GRASP_ATTACK_SECONDARY "mansus_grasp_attack_secondary"
```

Comment:
Here too, since we don't have secondary clicks at all atm

Reply:
Fixed